cmd/buildkitd/config/gcpolicy_unix.go

```diff
@@ -5,25 +5,18 @@ package config
 
 import (
 	"syscall"
-)
 
-func DetectDefaultGCCap() DiskSpace {
-	return DiskSpace{Percentage: 10}
-}
+	"github.com/moby/buildkit/util/bklog"
+)
 
-func (d DiskSpace) AsBytes(root string) int64 {
-	if d.Bytes != 0 {
-		return d.Bytes
-	}
-	if d.Percentage == 0 {
-		return 0
-	}
+var DiskSpacePercentage int64 = 20
```

Comment:
Don't change the percentage for unix (at least for the commit for v0.13.x)

Reply:
my bad, meant to change only for Windows, fixing.